npm-package/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Could you clarify what babel is being used for? I believe jest has support for typescript